Grove_LED_Bar.cpp

```diff
@@ -32,12 +32,13 @@
 
 #include "Grove_LED_Bar.h"
 
-Grove_LED_Bar::Grove_LED_Bar(unsigned char pinClock, unsigned char pinData, bool greenToRed, unsigned char led_num)
+Grove_LED_Bar::Grove_LED_Bar(uint8_t pinClock, uint8_t pinData, bool greenToRed, LedType type)
```

Comment:
[unsigned char led_num] was replaced with [LedType type], because we removed the macro definition MY9221_LED_NUM from the header file. Internally, we need to distinguish device types, so we converted this parameter into LedType for identifying device types.